e2e/k8s/vault-config/configure-vault.sh

```diff
@@ -91,7 +91,8 @@ vault write auth/myjwtk8s/role/external-secrets-operator \
 vault auth enable -path=mykubernetes kubernetes
 vault write auth/mykubernetes/config \
     kubernetes_host=https://kubernetes.default.svc.cluster.local \
-    kubernetes_ca_cert=@/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    kubernetes_ca_cert=@/var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+    issuer=https://kubernetes.default.svc.cluster.local
```

Comment:
we're still using an old version <1.9 of vault in our e2e tests, hence we gotta fix the `issuer` validation